pkg/agent/openflow/pipeline.go

```diff
@@ -2942,6 +2949,15 @@ func (f *featureService) gatewaySNATFlows() []binding.Flow {
 	return flows
 }
 
+func (f *featureService) isLocalServiceClusterIP(ip net.IP) bool {
+	for _, cidr := range f.serviceCIDRs {
```

Comment:
`serviceCIDRs` can be only used when AntreaProxy is disabled. We need another way to get the Service CIDRs.

Reply:
There is a way to find serviceCIDR automatically in MC controller, but I feel we should have one single source of truth for agent part. I believe reusing the `serviceCIDRs` config item and refine the comment is one way, not sure if there is any other way to share this kind of information between agents.

Reply:
I added a new field `serviceCIDR` under `multicluster` config section.